src/settings.rs

```diff
@@ -101,6 +106,9 @@ static DEFAULT_EXCLUDE: Lazy<Vec<FilePattern>> = Lazy::new(|| {
     ]
 });
 
+static DEFAULT_DUMMY_VARIABLE_RGX: Lazy<Regex> =
+    Lazy::new(|| Regex::new("_+$|_[a-zA-Z0-9_]*[a-zA-Z0-9]+?$").unwrap());
```

Comment:
```suggestion
    Lazy::new(|| Regex::new("^_$").unwrap());
```

Should we align with flake8?